index.js

```diff
@@ -37,7 +38,8 @@ class Rewt {
    *   @param {String} options.redisConn (required) A connection to redis.
    *   @param {String} options.redisNamespace (optional) The namespace to use.
    *   @param {Number} options.ttl (optional) The key's TTL before being rotated.
-   *
+   *   @param {Boolean} options.cacheSecret (optional) Whether or not to cache
```

Comment:
Are you thinking of times other than during rollout where we wouldn't want to cache this data?

Reply:
I think some consumers of this module may not want this behaviour, so I want it to be opt in.